horreum-web/src/domain/runs/TestDatasets.tsx

```diff
@@ -208,10 +211,33 @@ export default function TestDatasets() {
 
     const labelsSource = useCallback(() => testApi.filteringLabelValues(testIdInt), [testIdInt, teams, token])
 
-    return (
-        <>
-            <Toolbar className="pf-v5-u-justify-content-space-between" style={{ width: "100%" }}>
-                <ToolbarGroup>
+    const arrayOfClearCallbacks : any[] = [];
+    const clearCallback = (callback: () => void) => {
+        arrayOfClearCallbacks.push(callback);
+    }
+
+    // TODO: clear all filter cards
```

Comment:
Is this still a valid `TODO`?

Reply:
No, I have implemented this, I will remove the comment